targets/XPlatCppSdk/source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp

```diff
@@ -90,6 +90,12 @@ namespace PlayFab
         } // UNLOCK httpRequestMutex
     }
 
+    void PlayFabIXHR2HttpPlugin::SimplePostCall(std::string fullUrl, std::string requestBody, std::function<void(int, std::string)> successCallback, std::function<void(std::string)> errorCallback)
+    {
+        // TODO: This is currently only used by the GSDK, which does not run on Xbox
```

Comment:
The IXHR2 plugin is not part of the XPlatWindows or Linux project so it would not be compiled with for the gsdk. I would assume we do not need to add the function.

Reply:
If we don't, I think we'll get a compiler error that this class doesn't fully implement the IPlayFabHttpPlugin interface, right?

---
In reply to: [256922822](https://github.com/PlayFab/SDKGenerator/pull/402#discussion_r256922822) [](ancestors = 256922822)

Reply:
Got it, missed that change :)